src/highlight/highlightUtil.ts

```diff
@@ -18,6 +18,14 @@ export function getHighlightArea(shapes: Rect[]): number {
     return shapes.reduce((area, { height, width }) => area + height * width, 0);
 }
 
+export function getSelectionRange(): Range | null {
```

Comment:
Does this need to be a utility or can we encapsulate it inside the component where it's used?